samples/bookstore.yaml

```diff
@@ -0,0 +1,55 @@
+name: Autogenerated
+services:
+  7b24589e-2602-46d2-b82e-c0df9d3618d4:
```

Comment:
Please correct if I'm wrong. Where does these random strings come from? Following this example: https://github.com/service-mesh-patterns/service-mesh-patterns/blob/master/samples/bookInfoPattern.yaml, it seems we should set a fix name here.

Reply:
These random strings come from piecing together the k8s manifests into a pattern file. I think the logic is to simply replace any missing names with a random alphanumeric string